src/components/dapps/ContractItem.vue

```diff
@@ -79,25 +65,28 @@
           />
         </div>
       </div>
+
       <div>
-        <div class="text-xs text-gray-500 dark:text-darkGray-400">
-          Code bundle name
-        </div>
-        <div class="text-xs text-blue-900 dark:text-darkGray-100">
-          {{ contract.abi.json.contract.name }}
-        </div>
-      </div>
-      <div>
-        <div class="text-xs text-gray-500 dark:text-darkGray-400">
-          Contract ABI
-        </div>
+        <div class="text-xs text-gray-500 dark:text-darkGray-400">Messages</div>
         <div class="text-xs text-blue-900 dark:text-darkGray-100">
-          {{ shortenAbi }}
+          <message
```

Comment:
![bandicam 2021-06-30 14-05-05-972](https://user-images.githubusercontent.com/40356749/123905202-d836bb80-d9ac-11eb-86f6-3baac05e0517.jpg)
The contract message tooltip seems to cut off when it spans outside of the contract item component.
Related to this line https://github.com/PlasmNetwork/plasm-apps-vue/blob/45381bafb72eabba103bff0cae0b12a9dba092c2/src/components/dapps/Message.vue#L17

Reply:
Resolved